esp32_s3_eye/Kconfig

```
@@ -25,6 +25,33 @@ menu "Board Support Package"
             default 400000 if BSP_I2C_FAST_MODE
             default 100000
     endmenu
+
+    menu "SPIFFS - Virtual File System"
+        config BSP_SPIFFS_FORMAT_ON_MOUNT_FAIL
+            bool "Format SPIFFS if mounting fails"
+            default n
+            help
+                Format SPIFFS if it fails to mount the filesystem.
+
+        config BSP_SPIFFS_MOUNT_POINT
+            string "SPIFFS mount point"
+            default "/spiffs"
+            help
+                Mount point of SPIFFS in the Virtual File System.
+
+        config BSP_SPIFFS_PARTITION_LABEL
+            string "Partition label of SPIFFS"
+            default "storage"
+            help
+                Partition label which stores SPIFFS.
+
+        config BSP_SPIFFS_MAX_FILES
+            int "Max files supported for SPIFFS VFS"
```

```suggestion
            int "Maximum files that could be open at the same time"
```

If I remember correctly, every 1 increase in this amount consumes 10KB more memory... we better give a smaller default value I think (e.g. 2)